crates/ty_python_semantic/resources/mdtest/narrow/type_guards.md

```diff
@@ -17,16 +17,17 @@ def _(
     e: TypeGuard,  # error: [invalid-type-form]
     f: TypeIs,  # error: [invalid-type-form]
 ):
-    # TODO: Should be `TypeGuard[str]`
-    reveal_type(a)  # revealed: @Todo(`TypeGuard[]` special form)
+    reveal_type(a)  # revealed: TypeGuard[str]
     reveal_type(b)  # revealed: TypeIs[str | int]
-    # TODO: Should be `TypeGuard[complex & ~int & ~float]`
-    reveal_type(c)  # revealed: @Todo(`TypeGuard[]` special form)
+    # not `TypeGuard[complex & ~int & ~float]`: `complex` in argument position
+    # means `complex & int & float` semantically so `Intersection[complex,
```

Comment:
I think you mean 
```suggestion
    # means `complex | int | float` semantically so `Intersection[complex,
```

But honestly I would just use a different type above and skip all this complex/int/float nonsense, it's not germane